unity-renderer/Assets/UIComponents/Scripts/Components/SectionSelector/SectionSelectorComponentModel.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+[Serializable]
+public class SectionSelectorComponentModel
+{
+    public List<SectionToggleModel> sections;
+}
+
+[Serializable]
+public class SectionToggleModel
+{
+    public Sprite icon;
+    public string title;
+    public Toggle.ToggleEvent onSelectEvent;
```

Comment:
Isn't `Event` a bit redundant?  `onSelect` is not clear enough?

Reply:
Yep, you're right, it is a bit redundant. I have changed it in this place and in other ones where I was following the same naming, thanks! Done in: https://github.com/decentraland/unity-renderer/pull/1175/commits/3a0218e236f7309d2364fac1b463d548be90b174